lib/dotiw/action_view/helpers/date_helper.rb

```diff
@@ -9,15 +9,25 @@ module DateHelper
       include DOTIW::Methods
 
       def distance_of_time_in_words(from_time, to_time = 0, include_seconds_or_options = {}, options = {})
-        return _distance_of_time_in_words(from_time, to_time, options) if options[:vague]
+        if include_seconds_or_options.is_a?(Hash)
+          options = include_seconds_or_options
+        else
+          options[:include_seconds] ||= !!include_seconds_or_options
```

Comment:
This modifies the incoming hash, which is bad (TM). Do `options = options.merge(...)` here.